vllm/attention/backends/flashinfer.py

```diff
@@ -488,6 +488,14 @@ def advance_step(self,
 class FlashInferMetadataBuilder(AttentionMetadataBuilder[FlashInferMetadata]):
 
     def __init__(self, input_builder: "ModelInputForGPUBuilder"):
+
+        self.input_builder = input_builder
+        self.runner = input_builder.runner
+
+        self.sliding_window = input_builder.sliding_window
+        self.block_size = input_builder.block_size
```

Comment:
We probably need sm_scale too, but can go with the Flashinfer PR.

Reply:
yep, this PR just makes it easier to add more values in the builder. we can add them in the flashinfer pr.